concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md

```diff
@@ -159,4 +236,88 @@ The main idea is to filter the data source by the `parentID` for different drill
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :data-source="dxDataSource">
+            <DxSeries
+                argument-field="arg"
+                value-field="val"
+                type="bar"
+            />
+        </DxChart>
+    </template>
+
+    <script>
+    import DxChart, {
+        DxSeries
+    } from 'devextreme-vue/chart';
+    import DataSource from 'devextreme/data/data_source';
+
+    const population = [
+        // ...
+    ];
+
+    export default {
+        components: {
+            DxChart,
+            DxSeries
+        },
+        data() {
+            return {
+                dxDataSource: new DataSource({
+                    store: {
+                        type: 'array',
+                        data: population
+                    },
+                    filter: ['parentID', '=', '']
+                })
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Chart, {
+        Series
+    } from 'devextreme-react/chart';
+    import DataSource from 'devextreme/data/data_source';
+
+    const population = [
+        // ...
+    ];
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+             this.state = {
```

Comment:
I suggest removing state here as in [this case](https://github.com/DevExpress/devextreme-documentation/pull/751#discussion_r424449261).

Reply:
Fixed